test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -66,6 +66,27 @@ const filename = tmpdir.resolve('sync-write-stream.txt');
   assert.strictEqual(stream.fd, null);
 }
 
+// Verify behavior of destroy() when already destroy()ed
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd);
+
+  stream.on('close', common.mustCall());
+  assert.strictEqual(stream.destroy(), stream);
+  assert.strictEqual(stream.destroy(), stream);
+}
+
+// Verify that the file is not closed when autoClose=false
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd, { autoClose: false });
+
+  assert.strictEqual(stream.destroy(), stream);
+  fs.fstatSync(fd); // Does not throw
+  stream.on('close', common.mustCall());
+  fs.closeSync(fd);
+}
```

Comment:
Just a nit, feel free to ignore.

```suggestion
{
  const fd = fs.openSync(filename, 'w');
  const stream = new SyncWriteStream(fd, { autoClose: false });

  stream.on('close', common.mustCall());

  assert.strictEqual(stream.destroy(), stream);
  fs.fstatSync(fd); // Does not throw
  fs.closeSync(fd);
}
```